cypress/integration/Crosscutting/PageOptionsTest.feature

```diff
@@ -0,0 +1,104 @@
+            | /news-events/press-releases/2019                                                            | mobile     | none,Print         |
+            | /espanol/noticias/comunicados-de-prensa/2019                                                | tablet     | none,Print         |
+            | /about-nci/organization/nci-at-a-glance                                                     | mobile     | none,Print         |
+            | /espanol/infografia-nci                                                                     | tablet     | none,Print         |
+            | /about-cancer/coping/feelings/relaxation/hedge-maze                                         | mobile     | none,Print         |
+
+    Scenario Outline: Verify that font resizer button exists on the pages
+        Given user is navigating to "<url>"
+        Then page options are displayed
+        And "<pageOption2>" button is displayed
+        Examples:
+            | url                                                                                         | pageOption2  |
+            | /about-cancer/coping/feelings/relaxation                                                    | font resizer |
+            | /espanol/hoja-informativa-vitamina-d                                                        | font resizer |
+            | /about-cancer/coping/feelings/relaxation/loukissas-jen-biography                            | font resizer |
+            | /news-events/cancer-currents-blog/2019/hpv-vaccine-presidents-cancer-panel-improving-uptake | font resizer |
+            | /espanol/noticias/temas-y-relatos-blog/2017/tobacco-corrective-es                           | font resizer |
+            | /types/breast/research                                                                      | font resizer |
+            | /espanol/tipos/seno/investigacion                                                           | font resizer |
+            | /about-cancer/coping/feelings/relaxation/dfharvard                                          | font resizer |
+            | /about-cancer/coping/feelings/relaxation/2019-investigators-site                            | font resizer |
+            | /news-events/press-releases/2019/annual-report-nation-2018                                  | font resizer |
+            | /espanol/noticias/comunicados-de-prensa/2019/informe-anual-a-la-nacion-1975-2014            | font resizer |
+            | /about-cancer/treatment/side-effects/fatigue/fatigue-pdq                                    | font resizer |
+            | /espanol/cancer/tratamiento/efectos-secundarios/fatiga/fatiga-pdq                           | font resizer |
+
+    Scenario Outline: Verify that font resizer button does not exist on the pages where it should not
+        Given user is navigating to "<url>"
+        Then page options are displayed
+        And "<pageOption3>" button does not exist
```

Comment:
same as above - just pass font resizer directly
(also please be consistent with casing - either use lower or camel case in both scenarios)

Reply:
Done